main.go

```diff
@@ -2,26 +2,60 @@ package main
 
 import (
 	"log"
-	"os"
 
+	"github.com/alecthomas/kong"
 	"github.com/bobuhiro11/gokvm/flag"
 	"github.com/bobuhiro11/gokvm/probe"
 	"github.com/bobuhiro11/gokvm/vmm"
 )
 
-func main() {
-	// This line break is required by golangci-lint but
-	// such breaks are considered an anti-pattern
-	// at Google.
-	c, err := flag.ParseArgs(os.Args)
+type context struct {
+	Debug bool
+}
+
+type startCmd struct {
+	Kernel     string `arg:"" name:"kernel" help:"Path to linux kernel or EDK2/CloudHv firmware image" type:"path"`
```

Comment:
I prefer to use "flag" instead of "arg" for this as well.

Reply:
Hmmm. I choose 'arg' because it is required by default. 'flag' can be optional with a default value and I think it should be required without default value.

My idea is to use the kernel arg for firmware image as well and avoid to an additional arg/flag.

Reply:
done